internal_displacement/interpreter.py

```diff
@@ -443,37 +469,44 @@ def extract_dates(self, sentence, story, root=None):
             block_dates = self.match_entities_in_block(
                 date_entities, contiguous_token_block)
 
-            likely_block_dates = self.date_likelihood(block_dates, story)
+            # Filter unlikely dates from possible dates
+            likely_block_dates = self.date_likelihood(block_dates, publication_date)
             if likely_block_dates:
-                return self.convert_to_facts(likely_block_dates, "date", sentence[0].idx)
+                return likely_block_dates
```

Comment:
The reason for the returning extracted entities as Facts is to have a wrapper that can keep track of multiple things at the same time:
1. The underlying text
2. The token or tokens associated with that text
3. The location of the tokens within the text